packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -251,7 +351,7 @@ class ActionListener extends StatefulWidget {
     required this.listener,
     required this.action,
     required this.child,
-  })  : assert(listener != null),
+  })   : assert(listener != null),
```

Comment:
All of the `assert`s should be aligned if we're following the Flutter style guide.